src/clarity-angular/dropdown/dropdown-menu.ts

```diff
@@ -4,6 +4,11 @@ import {Component} from "@angular/core";
     selector: "clr-dropdown-menu",
     template: `
         <ng-content></ng-content>
-    `
+    `,
+    host: {
+        "[class.dropdown-menu]" : "true",
+        "[class.open]" : "true" // always true in angular; it's the responsibility of containing component to show/hide
```

Comment:
why is the open class on the `dropdown-menu`? I thought it has to be on the `clr-dropdown`. if this relates to the sub menu then shouldn't we add this class only when the sub menu is open?